almost_tcp/packet_fixtures_test.py

```diff
@@ -12,7 +12,7 @@
 
 
 @given(packets=...)
-def test_stream_send_structured_receive(packets: List[Packet]):
+def DISABLED_test_stream_send_structured_receive(packets: List[Packet]):
```

Comment:
Why DISABLED? Worth a comment if this is intentional?

Reply:
Done in e95d5ca